lib/org/openpsa/invoices/invoice/pdf.php

```diff
@@ -33,17 +37,21 @@ public function get_attachment(bool $autocreate = false) : ?midcom_db_attachment
         return $this->render_and_attach();
     }
 
-    public function get_button_options() : array
+    public function get_button_options(string $kind = null) : array
     {
         if ($attachment = $this->get_attachment()) {
             if ($this->invoice->sent) {
-                $message = 'invoice has already been sent. should it be replaced?';
+                if ($kind == 'reminder') {
```

Comment:
I find it a bit annoying that we have those $kind switches everywhere now. Can't we say that the other (default) kind is invoice? and than build the string like this: ` $kind . ' has already been sent. should it be replaced?';`